francis/__main__.py

```diff
@@ -19,7 +23,9 @@ def cli():
 @click.command()
 @click.argument("data_path")
 @click.option("-d", "--data-folder", is_flag=True)
-def train(data_path, data_folder):
+@click.option("-s", "--show-model", is_flag=True)
+@click.option("-p", "--pre-process", is_flag=True)
```

Comment:
nice addition, also good default as much of the time we don't need to preprocess